packages/certificator/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "certificator",
+  "version": "1.0.0",
+  "description": "ACME Client for obtaining SSL Certificates from Let's Encrypt.",
+  "main": "index.js",
+  "directories": {
+    "lib": "lib"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "bin": {
+    "certificator": "./bin/index.js"
```

Comment:
All other package.json are  structured the same..
-  bin: 'cli.js'
-  files: [] are explicit (makes it so that only files defined are included in the package
-  license is ISC license